src/components/ebay-textbox/test/test.server.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable no-unused-expressions */
```

Comment:
Is this because of `to.be.true`? I would suggest removing this, and instead be consistent in your `expect` usage below. Use `expect(a).to.equal(b)`, rather than `to.eql()` or `.to.be`